Common subdirectories: quarto-cli-1.4.250.original/src/command and quarto-cli-1.4.250/src/command
Common subdirectories: quarto-cli-1.4.250.original/src/config and quarto-cli-1.4.250/src/config
Common subdirectories: quarto-cli-1.4.250.original/src/core and quarto-cli-1.4.250/src/core
diff -Nu quarto-cli-1.4.250.original/src/dev_import_map.json quarto-cli-1.4.250/src/dev_import_map.json
--- quarto-cli-1.4.250.original/src/dev_import_map.json	2023-07-22 08:39:00.000000000 +1200
+++ quarto-cli-1.4.250/src/dev_import_map.json	2023-07-22 10:18:23.820831240 +1200
@@ -30,7 +30,7 @@
     "deno_dom/": "./vendor/deno.land/x/deno_dom@v0.1.35-alpha/",
     "xmlp/": "./vendor/deno.land/x/xmlp@v0.2.8/",
     "xml/": "./vendor/deno.land/x/xml@2.1.1/",
-    "another_cookiejar/": "./vendor/deno.land/x/another_cookiejar@v4.1.4/",
+    "another_cookiejar/": "./vendor/deno.land/x/another_cookiejar@v5.0.3/",
     "semver/": "./vendor/deno.land/x/semver@v1.4.0/",
     "media_types/": "./vendor/deno.land/x/media_types@v2.10.1/",
     "dayjs/": "./vendor/cdn.skypack.dev/dayjs@1.8.21/",
@@ -125,7 +125,7 @@
     "crypto/mod.ts": "./vendor/deno.land/std@0.185.0/crypto/mod.ts",
     "encoding/hex.ts": "./vendor/deno.land/std@0.185.0/encoding/hex.ts",
     "cliffy/prompt/secret.ts": "./vendor/deno.land/x/cliffy@v0.25.4/prompt/secret.ts",
-    "another_cookiejar/mod.ts": "./vendor/deno.land/x/another_cookiejar@v4.1.4/mod.ts",
+    "another_cookiejar/mod.ts": "./vendor/deno.land/x/another_cookiejar@v5.0.3/mod.ts",
     "testing/asserts.ts": "./vendor/deno.land/std@0.185.0/testing/asserts.ts",
     "https://deno.land/std@0.161.0/fmt/colors.ts": "./vendor/deno.land/std@0.185.0/fmt/colors.ts",
     "https://deno.land/std@0.161.0/encoding/base64.ts": "./vendor/deno.land/std@0.185.0/encoding/base64.ts",
Common subdirectories: quarto-cli-1.4.250.original/src/execute and quarto-cli-1.4.250/src/execute
Common subdirectories: quarto-cli-1.4.250.original/src/extension and quarto-cli-1.4.250/src/extension
Common subdirectories: quarto-cli-1.4.250.original/src/format and quarto-cli-1.4.250/src/format
diff -Nu quarto-cli-1.4.250.original/src/import_map.json quarto-cli-1.4.250/src/import_map.json
--- quarto-cli-1.4.250.original/src/import_map.json	2023-07-22 08:39:00.000000000 +1200
+++ quarto-cli-1.4.250/src/import_map.json	2023-07-22 10:18:34.027442398 +1200
@@ -30,7 +30,7 @@
     "deno_dom/": "https://deno.land/x/deno_dom@v0.1.35-alpha/",
     "xmlp/": "https://deno.land/x/xmlp@v0.2.8/",
     "xml/": "https://deno.land/x/xml@2.1.1/",
-    "another_cookiejar/": "https://deno.land/x/another_cookiejar@v4.1.4/",
+    "another_cookiejar/": "https://deno.land/x/another_cookiejar@v5.0.3/",
     "semver/": "https://deno.land/x/semver@v1.4.0/",
     "media_types/": "https://deno.land/x/media_types@v2.10.1/",
     "dayjs/": "https://cdn.skypack.dev/dayjs@1.8.21/",
Common subdirectories: quarto-cli-1.4.250.original/src/preview and quarto-cli-1.4.250/src/preview
Common subdirectories: quarto-cli-1.4.250.original/src/project and quarto-cli-1.4.250/src/project
Common subdirectories: quarto-cli-1.4.250.original/src/publish and quarto-cli-1.4.250/src/publish
Common subdirectories: quarto-cli-1.4.250.original/src/quarto-core and quarto-cli-1.4.250/src/quarto-core
Common subdirectories: quarto-cli-1.4.250.original/src/render and quarto-cli-1.4.250/src/render
Common subdirectories: quarto-cli-1.4.250.original/src/resources and quarto-cli-1.4.250/src/resources
Common subdirectories: quarto-cli-1.4.250.original/src/tools and quarto-cli-1.4.250/src/tools
Common subdirectories: quarto-cli-1.4.250.original/src/vendor and quarto-cli-1.4.250/src/vendor
Common subdirectories: quarto-cli-1.4.250.original/src/webui and quarto-cli-1.4.250/src/webui
